src/ape/managers/chain.py

```diff
@@ -279,6 +279,7 @@ def poll_blocks(
                 has_yielded = True
                 latest_confirmed_block_number = confirmable_block_number
 
+            has_yielded = False
```

Comment:
What's this for?

Reply:
I just renamed it to `has_yielded_before_reorg` to help clarify.
Basically tracking if we yield blocks so that if a re-org happens, we can warn the user that we gave them some blocks that are not longer canonical